### PR TITLE
Calculated Metrics Endpoint User Guide

### PR DESCRIPTION
**Pull request by johnwight**
This change provides a user guide for the /calculatedmetrics endpoint.

## Description

Documentation for the Analtytics 2.0 /calculatedmetrics endpoint

## Motivation and Context

This change has been requested so the that the endpoint is usable for our 2.0 API consumers.

## How Has This Been Tested?

This will be reviewed by the SME

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Not all of these apply.

### CODE REVIEW

**Comment by johnwight** [comment]
Hyrum: please double check the sentence on includeTypes default response behavior. I think I got it correctly but it might be read a different way.

Also, regarding building a definition for a calculated metric, should the user be directed to the document on building a definition for a "segment?"

Thanks.

File: calculatedmetrics.md [hunk 1]
```diff
@@ -1,10 +1,287 @@
...
+* `definition`
+* `authorization`
+* `compatibility`
+* `categories`
+* `reportTimeAttribution`
+* `warning`
+
+## IncludeTypes
+
+The GET multiple `/calculatedmetrics` endpoint supports the query parameter `includeType`. This parameter alters the set of calculated metrics that are included in API responses. By default, the response with this parameter includes only the calculated metrics owned by the user making the request. By using this parameter, the following values are also possible:
```
Comment (hyrumt) [inline calculatedmetrics.md]: By default, the response (i.e. _without_ this parameter) includes only the calc metric owned by the user.

When using this parameter, the following _options_ are possible:

**Comment by hyrumt** [comment]
@johnwight Even though the concept of building a definition for a segment and a calculated metric is the same, the actual pieces that are used are totally different. I don't think there would be any value in sending the user to the instructions on building a segment definition when they want to build a calc metric definition.

File: calculatedmetrics.md [hunk 2]
```diff
@@ -1,10 +1,287 @@
...
+```
+
+## Retrieving a single calculated metric
+
+To retrieve a single calculated metric, include its `id` in the request.
+
+### Example request
+
+```
+curl -X GET “https://analytics.adobe.io/api/[company name]/calculatedmetrics/[calculated metrics id]?locale=en_US” -H “x-api-key: [oauth token]” -H “x-proxy-global-company-id: [company name]” -H “Authorization: Bearer [ims user token]” -H “Accept: application/json” -H “Content-Type: application/json”
```
Comment (hyrumt) [inline calculatedmetrics.md]: The path should have `[calculated metric id]` (singular). Also, I need to double check if the `x-proxy-global-company-id` expects a companyId or the companyName. That header name may be incorrect.

**Comment by hyrumt** [comment]
I added a section to the wiki on calculated metric functions. That should be added here as well.

**Comment by hyrumt** [comment]
Looks good to me.

**Comment by hyrumt** [comment]
+1

**Pull request merged**